clang/test/Driver/fp-accuracy.c

```diff
@@ -0,0 +1,20 @@
+// RUN: %clang -### -target x86_64 -ffp-accuracy=high -c %s 2>&1 \
+// RUN:   | FileCheck --check-prefix=CHECK-H %s
+
+// RUN: %clang -### -target x86_64 -ffp-accuracy=low -c %s 2>&1 \
+// RUN:   | FileCheck --check-prefix=CHECK-L %s
+
+// RUN: %clang -### -target x86_64 -ffp-accuracy=medium -c %s 2>&1 \
+// RUN:   | FileCheck --check-prefix=CHECK-M %s
+
+// RUN: %clang -### -target x86_64 -ffp-accuracy=low:sin,cos -c %s 2>&1 \
+// RUN:   | FileCheck --check-prefix=CHECK-FUNC-1 %s
+
+// RUN: %clang -### -target x86_64 -ffp-accuracy=low:sin,cos -ffp-accuracy=high:tan -c %s 2>&1 \
```

Comment:
I should also be able to combine a general accuracy requirement with a function-specific accuracy requirement. For instance "-ffp-accuracy=high -ffp-accuracy=low:sin"